cmd/scollector/collectors/collectors.go

```diff
@@ -150,6 +152,18 @@ func Search(s []string) []Collector {
 	return r
 }
 
+// Adds configured tag overrides to all matching collectors
+func AddTagOverrides(s []Collector, tagOverride []conf.TagOverride) {
+	for _, to := range tagOverride {
+		re := regexp.MustCompile(to.CollectorExpr)
```

Comment:
Instead of panicing on a bad regex, should return an error, and os.Exit cleanly from main